dir/path.go

```diff
@@ -1,88 +1,71 @@
 package dir
 
 import (
-	"errors"
-	"io/fs"
+	"os"
+	"path"
+	"path/filepath"
 )
 
-const (
-	// ConfigFile is the name of config file
-	ConfigFile = "config.json"
+var (
+	SystemConfigDir  string // Absolute path of system level {NOTATION_CONFIG}
+	SystemLibexecDir string // Absolute path of system level {NOTATION_LIBEXEC}
+	UserConfigDir    string // Absolute path of user level {NOTATION_CONFIG}
+	UserLibexecDir   string // Absolute path of user level {NOTATION_LIBEXEC}
+)
 
-	// LocalCertificateExtension defines the extension of the certificate files
+// The relative path to {NOTATION_CONFIG}
+const (
+	// notation is the directory name for notation configurations.
+	notation = "notation"
```

Comment:
Let's move this unexported const to a separated `const` block.

Reply:
Updated.